Gopkg.toml

```diff
@@ -43,8 +43,12 @@ required = ["golang.org/x/lint/golint"]
 
 [[constraint]]
   name = "go.opencensus.io"
-  version = "0.19.2"
+  version = "=0.21.0"
 
 [[constraint]]
   name = "contrib.go.opencensus.io/exporter/ocagent"
-  version = "0.4.5"
+  version = "=0.3.0"
+
+[[override]]
```

Comment:
What's the plan for this override? Remove in a future version?

Reply:
Yes, once it's safe for kubernetes to ingest v1.3 of protobuf.

Reply:
Sounds good. Thanks again, @jhendrixMSFT!